gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/task/KubernetesConfigViewTaskTest.java

```diff
@@ -35,7 +35,7 @@ class KubernetesConfigViewTaskTest {
   private TestKubernetesExtension extension;
 
   @BeforeEach
-  void setUp() throws IOException {
+  void setUp() {
```

Comment:
Now that `IOException` isn't used anywhere in the class anymore. You also need to remove `import java.io.IOException;` on line 16